example/Makefile

```diff
@@ -10,8 +10,17 @@ x.wasm: main.go $(SRCS)
 
 build: x.wasm
 
-run: build
+result.json: x.wasm
 	@echo "Running WASM module..."
 	@cat fn-call.json \
-		| BKY_AS_HOST="local-server" bky-as attest-fn-call \
-		| jq -r '.transitive_attested_function_call | "Output:\n\t\(.claims.output | @base64d | fromjson | tojson)\nLogs:\n\t\(.logs | @base64d)"'
+		| BKY_AS_HOST="local-server" bky-as attest-fn-call > result.json
+
+run: result.json
```

Comment:
An early version of PR 5 had some functionality to make the output a little more readable. Now seemed like a nice time to bring that back.  

Note that now we have `result.json` target, I broke up the recipes a little more, but I am happy to squash them so that it is closer to the original PR 5 proposal.